plugins/Goals/Archiver.php

```diff
@@ -424,5 +424,8 @@ public function aggregateMultipleReports()
                 $columnsAggregationOperation,
                 $columnsToRenameAfterAggregation = null,
                 $countRowsRecursive = array());
+
+        $this->getProcessor()->processDependentArchive('Goals', API::NEW_VISIT_SEGMENT);
```

Comment:
can you quickly help me out where the stats for those two segments are shown?

Reply:
here's the controller action: https://github.com/matomo-org/matomo/blob/3.x-dev/plugins/Goals/Controller.php#L89